webapp/channels/src/utils/notifications.ts

```diff
@@ -50,12 +50,8 @@ export function showNotification(
             icon = iconWS;
         }
 
-        if (!('Notification' in window)) {
-            throw new Error('Notification not supported');
-        }
-
-        if (typeof Notification.requestPermission !== 'function') {
-            throw new Error('Notification.requestPermission not supported');
+        if (!isNotificationAPISupported()) {
+            throw new Error('Notification API is not supported');
```

Comment:
Should this throw an error? I would expect it to log an error and allow the app to continue without interrupting the user with an error that 1) doesn't affect their usage of the app, and 2) can't be fixed by the user anyway.

Reply:
Throwing error here as two purpose, i.e `showNotification` function is used in two places in code base where in both its called in try/catch block so the parent functions are responsible to handling the catch cases. `notifyMe` & `showSessionExpiredNotificationIfNeeded` both have catch handlers. Second being there is no reason for them to continue to execute the rest of the function since rest of the function assuse Notification is not undefined object